docs/references/general/fitacf.md

```diff
@@ -1,2 +1,175 @@
+| *stid*                  | **None**         | ***short***  | Station identifier                                                                      |
+| *time.yr*               | **None**         | ***short***  | Year                                                                                    |
+| *time.mo*               | **None**         | ***short***  | Month number (not padded)                                                               |
+| *time.dy*               | **None**         | ***short***  | Day (not padded)                                                                        |
+| *time.hr*               | *hrs*          | ***short***  | Hour                                                                                    |
+| *time.mt*               | *min*          | ***short***  | Minute                                                                                  |
+| *time.sc*               | *s*        | ***short***  | Seconds                                                                                 |
+| *time.us*               | *us* | ***int***    | Micro-seconds                                                                           |
+| *txpow*                 | *kW*       | ***short***  | Transmitted power                                                                       |
+| *nave*                  | **None**         | ***short***  | Number of pulse sequences transmitted                                                   |
+| *atten*                 | **None**         | ***short***  | Attenuation level                                                                       |
+| *lagfr*                 | *us* | ***short***  | Lag to first range                                                                      |
+| *smsep*                 | *us* | ***short***  | Sample seperation                                                                       |
+| *ercod*                 | **None**         | ***short***  | Error Code                                                                              |
+| *stat.agc*              | **None**         | ***short***  | AGC status word                                                                         |
+| *stat.lopwr*            | **None**         | ***short***  | LOPWR status word                                                                       |
+| *noise.search*          | **None**         | ***float***  | Calculated noise from clear frequency search                                            |
+| *noise.mean*            | **None**         | ***float***  | Average noise from clear frequency band                                                 |
+| *channel*               | **None**         | ***short***  | Channel number for variant changes in radars parameters in one scan, ex) frequency      |
```

Comment:
```suggestion
| *channel*               | **None**         | ***short***  | Channel number, historically used to denote different Tx/Rx channels on Stereo radars, now also used to denote changes in radars parameters between scans, e.g. alternating between 2 frequencies scan to scan      |
```

Reply:
Again, slight rewording since there are several operational stereo radars
```suggestion
| *channel*               | **None**         | ***short***  | Channel number, used to denote different Tx/Rx channels on Stereo radars, and to denote changes in radar operating parameters between scans, e.g. alternating between 2 frequencies scan-to-scan      |
```

Reply:
Taking @ecbland